### PR TITLE
[APPAI-1411] [Component Analyses] Add more meta data about package vulnerability

### PR DESCRIPTION
**Pull request by dgpatelgit**
PR resolving Jira issue :: https://issues.redhat.com/browse/APPAI-1411

### CODE REVIEW

File: tests/utility/v2/test_ca_response_builder.py [hunk 1]
```diff
@@ -348,11 +385,45 @@ def test_get_cve_maps_non_empty(self):
...
+            title="Test title for CVE",
+            url="https://test.com/cve-01",
+            cve_ids=["CVE-01", "CVE-02", "CVE-03"])
+        ]
+        response_obj._cves = [vul_data]
+        cve_maps = response_obj.get_cve_maps()
+        self.assertListEqual(cve_maps, mocked_response)
+
+    def test_get_cve_maps_default(self):
+        """Test Severity with unknown value, raises exception."""
```
Comment (deepak1725) [inline tests/utility/v2/test_ca_response_builder.py:413]: Can we update function docs

**Comment by dgpatelgit** [comment]
> @dgpatelgit Please update the API spec.

This is done, PTAL

File: tests/utility/v2/test_ca_response_builder.py [hunk 2]
```diff
@@ -316,43 +318,115 @@ def test_get_severity_return_high(self):
...
+        """Test Severity Procedure. Severity "Medium" has higher precedence."""
+        response_obj = ComponentAnalysisResponseBuilder(self.eco, self.pkg, self.ver)
+        response_obj._cves = [
+            {'severity': ['low']},
+            {'severity': ['medium']},
+            {'severity': ['low']},
+            {'severity': ['low']},
+        ]
+        severity = response_obj.get_severity()
+        self.assertListEqual(severity, ['medium'])
```
[the remark this answers is not in the document]
Comment (dgpatelgit) [reply tests/utility/v2/test_ca_response_builder.py:339]: It is "highest" amount the given severity. We have separate test cases to verify for "Critical", "High" and "Medium"
"Low" severity and invalid severity test case were missing, to achieve 100% unit test coverage, I have added them, actually not connected to these changes.

[the remark this answers is not in the document]
Comment (arajkumar) [reply tests/utility/v2/test_ca_response_builder.py:339]: Okay.

**Pull request merged**